package.json

```diff
@@ -13,9 +13,27 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "precommit": "lint-staged"
```

Comment:
also add `prettier` to scripts and call it on `lint-staged`
example: `"prettier": "prettier --write --single-quote true --trailing-comma all --print-width 120",`